.github/workflows/update-configuration.yml

```diff
@@ -1,59 +1,17 @@
-name: "Update Configuration"
+name: "Update Configuration and Build"
 
 on:
   workflow_dispatch:
   push:
 
 jobs:
   update:
-    name: "Update Configuration in manifest.json"
+    name: "Update Configuration & Build"
     runs-on: ubuntu-latest
     permissions: write-all
 
     steps:
-      - uses: actions/checkout@v4
-
-      - name: Setup node
-        uses: actions/setup-node@v4
-        with:
-          node-version: "20.10.0"
-
-      - name: Install deps and run configuration update
-        run: |
-          yarn install --immutable --immutable-cache --check-cache
-          yarn tsc --noCheck --project tsconfig.json
-
-      - name: Update manifest configuration using GitHub Script
-        uses: actions/github-script@v7
-        with:
-          script: |
-            (async () => {
-              const fs = await import('fs/promises');
-              const path = await import('path');
-
-              const { pluginSettingsSchema } = await import("${{ github.workspace }}/src/types/plugin-inputs.js");
-
-              const manifestPath = path.resolve("${{ github.workspace }}", './manifest.json');
-              const manifest = JSON.parse(await fs.readFile(manifestPath, 'utf8'));
-
-              const configuration = JSON.stringify(pluginSettingsSchema);
-              manifest["configuration"] = JSON.parse(configuration);
-
-              const updatedManifest = JSON.stringify(manifest, null, 2);
-              console.log('Updated manifest:', updatedManifest);
-              await fs.writeFile(manifestPath, updatedManifest);
-            })();
-
-      - name: Commit and Push generated types
-        run: |
-          git config --global user.name 'ubiquity-os[bot]'
-          git config --global user.email 'ubiquity-os[bot]@users.noreply.github.com'
-          git add ./manifest.json
-          if [ -n "$(git diff-index --cached --name-only HEAD)" ]; then
-            git commit -m "chore: updated generated configuration" || echo "Lint-staged check failed"
-            git push origin HEAD:${{ github.ref_name }}
-          else
-            echo "No changes to commit"
-          fi
+      - uses: ubiquity-os/action-deploy-plugin@main
         env:
-          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          app-id: ${{ secrets.APP_ID }}
+          app-private-key: ${{ secrets.APP_PRIVATE_KEY }}
```

Comment:
Kebab case syntax? Is that right?

Reply:
It's usually prettier to use within Actions, but can be changed to camelCase if you want.

Reply:
Yeah I would consider these secret names to be standardized to `APP_ID` and `APP_PRIVATE_KEY` at this point